packages/orchestration/README.md

```diff
@@ -327,6 +327,23 @@ const response = await orchestrationClient.chatCompletion({
 return response.getContent();
 ```
 
+`buildDocumentGroundingConfig()` function provides convenience to create the document grounding service configuration using the minimal set of required values:
+
+```ts
+import {
+  buildDocumentGroundingConfig
+} from '@sap-ai-sdk/orchestration';
+
+const orchestrationConfig: {
+  ...,
+  grounding: buildDocumentGroundingConfig(
+    input_params: ['groundingInput'],
+    output_param: 'groundingOutput',
```

Comment:
[pp] I feel like this example isn't enough, as it leaves out how to configure the template, which isn't exactly obvious I'd say. Also, it's not intuitive what `'filter-id'` can/should be. If the ID is arbitrary, we may as well make it optional and create some ID for the user.

Reply:
The code example just above this (the Grounding section) has a detailed example (including templating). I just wanted to explain that the grounding part can be extracted to this function.

Reply:
Ah okay, that should be sufficient. Maybe I'd switch the order though.. if users follow the instructions from top-down they only later learn that they could have used something simpler for most cases